gomint-server/src/main/java/io/gomint/server/entity/animal/EntityCod.java

```diff
@@ -41,6 +45,30 @@ private void initEntity() {
         this.setHealth(3);
     }
 
+    @Override
+    protected void kill() {
+        super.kill();
+
+        if (deadTimer > 0) {
+            return;
+        }
+
+        // Drop items
+        Location location = this.getLocation();
+        this.world.dropItem(location, ItemCod.create(1));
+
+        ThreadLocalRandom random = ThreadLocalRandom.current();
+        if (random.nextInt(4) == 0) {
+            int amount = random.nextInt(1, 3);
+            this.world.dropItem(location, ItemBone.create(amount));
```

Comment:
Cod seems to drop more according to https://minecraft.gamepedia.com/Cod#Drops

Reply:
random.nextInt(1, 3) returns 1 or 2 (1 is included and 3 is excluded), it fits the `1–2 bones (25% chance)‌[Bedrock Edition only]`, but I actually missed this line in the changelog: `BE 1.16.100 Cod now drop the cooked version of their item drops, if killed while on fire.`, it was conflicting with https://minecraft.gamepedia.com/Drops#Mob_drops which says that cod only drops raw in BE, but now I think the Drops page is outdated, so it should drop the cooked version when it's on fire. This really needs to be fixed.